bioimageio/core/prediction_pipeline/_prediction_pipeline.py

```diff
@@ -59,6 +68,20 @@ def output_specs(self) -> List[OutputTensor]:
         """
         ...
 
+    @abc.abstractmethod
+    def load(self) -> None:
+        """
+        optional step: load model onto devices before calling forward if not using it as context manager
+        """
+        ...
+
+    @abc.abstractmethod
+    def unload(self) -> None:
+        """
+        free any device memory in use
+        """
+        ...
+
```

Comment:
there is basically a close() method here that I named unload() as load() makes more sense that open()...